### PR TITLE
Some SCIM and SAML fixes

### PR DESCRIPTION
**Pull request by zuluecho9**
Assorted fixes for user mgmt liaisonship. Work related to DOC-6834. I wouldn't spend much time thinking about the phrasing/readability, honestly; these docs are pretty messy and convoluted and have quite a few issues in this regard. Focused on just making them accurate for now.

### CODE REVIEW

File: src/content/docs/accounts/accounts/automated-user-management/onelogin-scimsso-application-configuration.mdx
Comment (barbnewrelic) [inline src/content/docs/accounts/accounts/automated-user-management/onelogin-scimsso-application-configuration.mdx:114]: Should this be an H2 ## rather than an H3 ###?

File: src/content/docs/accounts/accounts/automated-user-management/onelogin-scimsso-application-configuration.mdx
Comment (barbnewrelic) [inline src/content/docs/accounts/accounts/automated-user-management/onelogin-scimsso-application-configuration.mdx:129]: Should this be an H2 ## rather than an H3 ###?

**Review by barbnewrelic — APPROVED** [review]
I noted 2 questions in one of these six docs. As far as I could tell, everything else is fine. OK to merge after you take a look and fix as applicable, no need for another review. Thanks for all your hard work on this!

File: src/content/docs/accounts/accounts/automated-user-management/onelogin-scimsso-application-configuration.mdx
Comment (zuluecho9) [reply src/content/docs/accounts/accounts/automated-user-management/onelogin-scimsso-application-configuration.mdx:129]: it was a few nested h3s under an h2

**Pull request merged**
